guides/v2.1/javascript-dev-guide/widgets/widget_dialog.md

```diff
@@ -146,6 +157,43 @@ Beside default functionality, this method calls the `_mouseLeave()` and `_mouseE
 ### `close()` {#d_close}
 Beside default functionality, this method removes the classes from trigger and drop-down parent.Also clears the timeout if the latter exists.
 
+## Code sample
+
+The following example shows how to initialize the dropdown dialog widget and pass options during the initialization.
+This example uses some CSS classes that are used for minicart.
+
+```html
+<div data-block="dropdown" class="minicart-wrapper">
+    <button type="button" class="action" data-trigger="trigger">
+        <span data-bind="i18n: 'Click Here'"></span>
+    </button>
+</div>
+<div class="block block-minicart"
+     data-mage-init='{
+        "dropdownDialog": {
+            "appendTo": "[data-block=dropdown]",
+            "triggerTarget":"[data-trigger=trigger]",
+            "timeout": 2000,
+            "closeOnMouseLeave": false,
+            "closeOnEscape": true,
+            "autoOpen": true,
+            "triggerClass": "active",
+            "parentClass": "active",
+            "buttons": []
+        }
+     }'>
+    <div id="minicart-content-wrapper">
+        Here is our content
+    </div>
+</div>
+```
+
+### Result
+
+The result is a dropdown dialog and a button (_Click Here_) that opens the dialog.
+
+![DropdownDialog Widget]({{ site.baseurl }}/common/images/widget/dropdown-dialog-widget-result.png)
```

Comment:
According to this PR, you actually added the image to the following location:

```guides/v2.1/javascript-dev-guide/widgets/images/dropdown-dialog-widget-result.png```

Please move the image to:

```common/images/widget/dropdown-dialog-widget-result.png```